platformio.ini

```diff
@@ -300,7 +300,7 @@ board_build.partitions = ${esp32.default_partitions}
 
 [env:esp32_eth]
 board = esp32-poe
-platform = espressif32@2.0
+platform = espressif32@2.1
```

Comment:
Is this change necessary?
If yes: Why not updating to 3.4?

Reply:
Ugh, I didn't realize the PR was pulling in all the commits I made on the branch since I created it. I assumed it was frozen at the commit I based the PR on.

I've been using the branch for local dev on a specialized use case since then, so it's no longer in any state to merge. Reverting this PR, and I'll untangle it and open a new one later

Reply:
No problem. I do not know any of DMX stuff so was just curious about changes I understood. 😄